### ISSUE TITLE
Recommend to use imageio.v3 instead of imageio

### ISSUE THREAD

**Issue by xunmeibuyue** [Zulko/moviepy#1894]
I would like to recommend to use `imageio.v3` instead of `imageio` when just import `imread` from `imageio`, specifically in such two position:
https://github.com/Zulko/moviepy/blob/858bb81fba7e09f1f562283ed6d1394db883b6c7/moviepy/video/VideoClip.py#L14
https://github.com/Zulko/moviepy/blob/0751d072d9b61be3466f39184beba004d61104f1/moviepy/video/io/ImageSequenceClip.py#L8

This is because `imageio.v3.imread` support more image format than `imageio.imread` such as `.webp`. And if necessary I would like to submit a PR.

**Comment by mgaitan** [comment]
+1 to this. If you still can submit a PR I could review quickly and move it forward. Thanks.

**Comment by xunmeibuyue** [comment]
Of course! I have submitted a PR at https://github.com/Zulko/moviepy/pull/1955#issue-1679868726. Thanks.

**Comment by xunmeibuyue** [comment]
A new PR has been submitted at https://github.com/Zulko/moviepy/pull/1956. Thanks.
